apps/ai-content-generator/src/components/app/dialog/common-generator/CommonGenerator.styles.tsx

```diff
@@ -1,6 +1,6 @@
-import { css } from 'emotion';
 import tokens from '@contentful/f36-tokens';
 import { DIALOG_MIN_HEIGHT } from '@configs/dialog/dialogConfig';
+import { css } from '@emotion/react';
```

Comment:
was this causing the build issue?

Reply:
https://github.com/contentful/apps/pull/4312/files#diff-418e2acaa9a45cf595ea8d75757b268959cb85043a352a758f3aecb775830e19R14
This line in the end fixed the build issue. But I could only get this working with `@emotion/react`